project/plugins.sbt

```diff
@@ -5,3 +5,4 @@ addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.1.2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.1")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
 addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.12.0")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0") // "2.4.0" is just sbt plugin version
```

Comment:
I think we should use 3.0.0-RC5

Reply:
That's the sbt plugin, not the actual scalafmt version:

https://github.com/scalameta/sbt-scalafmt/tags